src/client.rs

```diff
@@ -48,6 +50,7 @@ impl Client {
             signer,
             proxy_addr,
             force_proxy,
```

Comment:
We path both proxy and proxy enforce flat to `netservices::Session::connect` methods!